Foundation/FileManager.swift

```diff
@@ -1450,7 +1450,7 @@ open class FileManager : NSObject {
 #if os(Windows)
         NSUnimplemented()
 #else
-        if rmdir(path) == 0 {
+        if _fileSystemRepresentation(withPath: path, { rmdir($0) }) == 0 {
```

Comment:
I think that it would be better to wrap the path conversion at the `_removeItem` call site.